data/maps/Route2/scripts.pory

```diff
@@ -28,11 +28,10 @@ script Route2_EventScript_CeliaDad {
 }
 
 script Route2_EventScript_RouteSign {
-	msgbox("TRAINER TIPS\pYour POKéDEX will tell you how\nmany POKéMON are available to you.\lIt even updates automatically!", MSGBOX_DEFAULT)
+	lock
```

Comment:
If you're writing your own `lock`/`release`, you should use the default msgbox instead of the sign one or you'll double `lock`/`release` which can cause issues. Also, for signs you should use `lockall` instead of `lock`.